.pre-commit-config.yaml

```
@@ -189,6 +189,8 @@ repos:
     additional_dependencies:
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
     - pytest
+    - requests  # dep of `credentials.aim`, `credentials.centrify_vault`,
+    # `credentials.conjur` & `credentials.hashivault`
```

I've never seen comments added below lines they refer to. It's always either above or on the same line. The same line should be preferred, since it guards against new things appearing between the comment and the code, making them drift away from each other over time. The preceding line is the last resort in cases where we can't satisfy the line length expectations.
```suggestion
    # The following dependency is used in `credentials.aim`,
    # `credentials.centrify_vault`, `credentials.conjur`
    # and `credentials.hashivault`:
    - requests
```